spec/atom-environment-spec.coffee

```diff
@@ -174,6 +174,19 @@ describe "AtomEnvironment", ->
       atom.loadStateSync()
       expect(atom.state.stuff).toBe("cool")
 
+    it "saves state on keypress and mousedown events", ->
+      spyOn(atom, 'saveStateSync')
+
+      keypress = new KeyboardEvent('keypress')
+      atom.document.dispatchEvent(keypress)
+      advanceClock 1100
+      expect(atom.saveStateSync).toHaveBeenCalled()
+
+      mousedown = new MouseEvent('mousedown')
+      atom.document.dispatchEvent(mousedown)
+      advanceClock 1100
+      expect(atom.saveStateSync).toHaveBeenCalled()
+
```

Comment:
Spy not working as expected and causing test to fail:

<img width="794" alt="screen shot 2016-01-27 at 11 18 09 am" src="https://cloud.githubusercontent.com/assets/7910250/12623578/0f524114-c4e8-11e5-8ffe-a0d7a774c6cc.png">